website/docs/reference/resource-properties/constraints.md

```diff
@@ -350,7 +350,61 @@ models:
 
 </File>
 
-Expected DDL to enforce constraints:
+### Column-level constraint on nested column:
+
+<File name='models/nested_column_constraints_example.sql'>
+
+```sql
+{{
+  config(
+    materialized = "table"
+  )
+}}
+
+select
+  'string' as a,
```

Comment:
It would be nice if this example were more "real world" but this gets the point across.